text/2-aurgy.md

```diff
@@ -0,0 +1,135 @@
+---
+rfc pr: [#4](https://github.com/UCLA-Creative-Labs/cl-rfcs/pull/4)
+tracking issue: https://github.com/UCLA-Creative-Labs/cl-rfcs/issues/2
+---
+
+# Aurgy
+
+A web app that takes a group of users and a theme and creates a playlist
+of songs which match the group's collective tastes and the theme.
+
+## Blurb
+
+This project, codenamed _Aurgy_, is intended to take Spotify's new (beta) feature,
+Blends, and extend it to groups. Additionally, we want to allow for choosing an
+overall theme for the generated playlist; so that a single group could perhaps
+generate multiple playlists for different ocassions.
+
+The project is meant for, well, pretty much anyone who listens to music. A PWA
+would be ideal for such a project, since it will include a means of forming
+groups, choosing a theme, and playing the generated playlist.
+
+### Future goals
+
+* This project could additionally just become a means to form a collaborative queue,
+  where users can freely add and remove songs to a queue from their own phones.
+* It would be really cool if the user could type in, in their own words, a desired theme,
+  and using NLP or something along those lines we could figure out what a playlist would
+  look like. [This article is very interesting] (thanks @sriramb2000).
+
+[This article is very interesting]: https://github.com/UCLA-Creative-Labs/cl-rfcs/issues/2
+
+## Why are we doing this?
+
+Spotify's Blends are cool, but they're still limited. Being able to listen to
+things as a group would be a great feature to have. Also, being able to generate
+group playlists based on themes would be great to have for all kinds of events
+and occasions---saves everyone the trouble of passing the phone around and searching
+up songs.
+
+## What is the high level design/implementation?
+
+The best plan of attack for this project would be to create a PWA. Realistically, this
+would be a mobile-first application, but by creating a PWA we can a) support desktop
+and b) avoid working with something like React Native (bleh) but still achieve cross-
+platform support.
+
+This project has a few key components:
+* Creating and managing a group/lobby
+* Generating a playlist
+* Playing music from the playlist
+
+Let's delve into these in more detail.
+
+#### Creating and managing a group/lobby
+
+In this project, users should be able to join a private, protected group. We could probably
+accomplish this using a standard lobby system: one person acts as the host and creates the
```

Comment:
not sure if we need a host? I think something like a UID link is a good idea, but we should be the 'host'

Reply:
Well I figure by having a "host", we can let them control who can join the group. It's like the driver deciding who they let in their car on the road trip.